lib/config/configuration.js

```diff
@@ -949,6 +949,8 @@ Configuration.prototype.registerDefaultRules = function() {
     this.registerRule(require('../rules/require-matching-function-name'));
 
     this.registerRule(require('../rules/disallow-space-before-semicolon'));
+
+    this.registerRule(require('../rules/validate-order-in-object-keys'));
```

Comment:
@mrjoelkemp what about sorting these `registerRule` calls? I had some difficulty figuring out if I should just add the new rule to the bottom.